### PR TITLE
Marginally improve pytorch interface

### PR DESCRIPTION
**Pull request by half-potato**
I want to use from_pytorch to allow pytorch to handle memory allocation but I don't want to allocate unecessary memory for the gradient. This also handles the two dtypes allowed logically.

### CODE REVIEW

**Comment by mmacklin** [comment]
Hi @half-potato, we recently addressed this issue in `0.8.0` so that you can override the gradient allocation in `wp.from_torch()` by setting `requires_grad=False`.

Hope that addresses your problem - if not feel free to open an issue.

Best,
Miles
